### PR TITLE
Schema docs, second round

### PR DESCRIPTION
**Pull request by djmitche**
Based on comments in #72

### CODE REVIEW

File: cis/plugins/validation/schema.json [hunk 1]
```diff
@@ -178,39 +205,61 @@
...
             "type": "array",
-            "description": "All PGP fingerprints associated with this user's profile such as `bob` or `mary`. Order is undefined.",
+            "description": "All PGP fingerprints associated with this user's profile such as `D8097934A92E4B4210368102FF8B7AC6154E3226`. Order is undefined.",
             "items": {
                 "type": "object",
                 "properties": {
                     "value": {
-                        "type": "string"
+                        "type": "string",
+                        "description": "The key, as a hex string"
```
Comment (gdestuynder) [inline cis/plugins/validation/schema.json]: The full fingerprint as a hex string

File: cis/plugins/validation/schema.json [hunk 2]
```diff
@@ -243,7 +292,7 @@
                     "lastUsed": {
                         "type": "string",
                         "format": "date-time",
-                        "description": "Date at which this group membership last granted access to a resource (generally, an RP) for the user. This may be used to automatically expire and deny access when the lastUsed attribute is older than 3 month, for example."
+                        "description": "Date at which this group membership last granted access to a resource (generally, an RP) for the user. This may be used to automatically expire and deny access when the lastUsed attribute is older than 3 month, for example. This property is updated automatically."
```
Comment (gdestuynder) [inline cis/plugins/validation/schema.json]: I suspect this addition might be confusing (all fields may be updated automatically, though some initially usually come from the user)

File: docs/Profiles.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+schema](http://json-schema.org/).  The schema contains descriptions of its
+fields and is the primary reference for the content of a profile.  It is
+available at
+[cis/plugins/validation/schema.json](/cis/plugins/validation/schema.json).
+
+A [sample test profile](/tests/data/profile-good.json) is also available,
+although it omits many optional fields.
+
+## Groups
+
```
Comment (gdestuynder) [inline docs/Profiles.md]: I think this needs a picture and some rewording but would merge it as is for now, until we improve it

File: docs/Profiles.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+The profile contains both `groups` and `authoritativeGroups`. The difference is
+subtle and it can be difficult to determine what an RP (relying party) should
+use for purposes such as access control.
+
+Access providers is expected to maintain and enforce `authoritativeGroups`.
```
Comment (gdestuynder) [inline docs/Profiles.md]: s/is/are/

**Comment by gdestuynder** [comment]
Tests will need fix: https://travis-ci.org/mozilla-iam/cis/builds/265764152#L487

**Comment by djmitche** [comment]
Comments addressed and I *think* the tests are fixed.  I'm using py3.5 and getting some str/bytes issues locally, but I hope those are not an issue in Travis.

**Comment by djmitche** [comment]
😌 tests pass

File: tests/data/profile-bad.json [hunk 5]
```diff
@@ -19,6 +19,7 @@
 		{
 			"value": "leomcardle@gmail.com",
 			"verified": true,
+            "primary": false,
```
Comment (gdestuynder) [inline tests/data/profile-bad.json]: seems to be a tab vs spaces issue :(

File: tests/data/profile-good.json [hunk 6]
```diff
@@ -20,6 +20,7 @@
 		{
 			"value": "leomcardle@gmail.com",
 			"verified": true,
+            "primary": false,
```
Comment (gdestuynder) [inline tests/data/profile-good.json]: same
feel free to reflow as spaces btw.

**Comment by djmitche** [comment]
@gdestuynder I think this is ready to go?

**Pull request merged**